src/components/projectDetail/ProjectTabs.tsx

```
@@ -315,12 +340,11 @@ const ProjectTabs = ({ projectData, isLoading, error }: ProjectTabsProps) => {
                               {/* 리뷰 작성자 프로필 사진 */}
                               <div className="h-16 w-16 overflow-hidden rounded-full">
                                 <img
-                                  src={review.imageUrl}
+                                  src={review.imageUrl || "/placeholder.svg"}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)